figures/mapping-histology-labels.Rmd

```diff
@@ -301,25 +376,26 @@ broad_histology_order_df <- histologies_df %>%
   filter(sample_type == "Tumor",
          # Only count histologies that we'll have a hex code for
          broad_histology %in% included_labels_df$broad_histology) %>%
-  # Filter to unique sample--broad_histology pairs
+  # Need to add broad_histology_display here because that's what we need to 
+  # order/count
+  inner_join(select(palette_df,
+                    broad_histology,
+                    broad_histology_display)) %>%
+  # Filter to unique sample--broad_histology_display pairs
   select(sample_id, 
-         broad_histology) %>%
+         broad_histology_display) %>%
   distinct() %>%
   # Count samples within a broad histology
-  count(broad_histology) %>%
-  # Add Other placeholder
-  bind_rows(data.frame(broad_histology = "Other",
-                       n = 0, 
-                       stringsAsFactors = FALSE)) %>%
+  count(broad_histology_display) %>%
   # Reorder based on sample size except Benign tumor and Other should come last
   # And then add numeric column with the order
-  mutate(broad_histology = forcats::fct_reorder(broad_histology,
-                                                n,
-                                                .desc = TRUE) %>%
+  mutate(broad_histology_display = forcats::fct_reorder(broad_histology_display,
+                                                        n,
+                                                        .desc = TRUE) %>%
            forcats::fct_relevel("Benign tumor",
                                 "Other",
                                 after = Inf),
```

Comment:
Something happened upstream where this isn't working and throws a warning, since the "Other" level isn't there - 
```
Warning: Problem while computing
`broad_histology_display = ... %>% ...`.
ℹ Unknown levels in `f`: Other
```

Though, I do see that "Other Tumor" _is_ a level here?

Reply:
Should be fixed now